config/application.rb

```diff
@@ -7,10 +7,10 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module RailsStarter
+module App2
```

Comment:
We should probably rename this back to `RailsStarter`. There are a couple other references too.